api.planx.uk/modules/webhooks/docs.yaml

```diff
@@ -253,7 +244,7 @@ components:
                 type: string
                 description: A message returned to the client
 paths:
-  /webhooks/hasura/sendSlackNotification:
+  /webhooks/hasura/send-slack-notification:
```

Comment:
This now matches the route - https://github.com/theopensystemslab/planx-new/blob/782f982894bdfd1d5853ef5c5e22991214743320/api.planx.uk/modules/webhooks/routes.ts#L39-L43